### ISSUE TITLE
kitchen_sink test possible incorrect opcodes

### ISSUE THREAD

**Issue by ghost** [WebAssembly/binaryen#138]
Comparing the binary output of the kitchen_sink test against the expect order of the operators suggests the following codes are wrong: a8 aa ab ad ae af b0 b1 b3 b4 b5

Sorry if I've put them in the wrong order, but my reading of the v8 spec is that these are:

```
    (f32.convert_s/i32 #xa8 (i32) f32)   ; f32.sconvert_i32
    (f32.convert_s/i64 #xaa (i64) f32)   ; f32.sconvert_i64
    (f32.convert_u/i64 #xab (i64) f32)   ; f32.uconvert_i64
    (f32.reinterpret/i32 #xad (i32) f32) ; f32.reinterpret_i32
    (f64.convert_s/i32 #xae (i32) f64)   ; f64.sconvert_i32
    (f64.convert_u/i32 #xaf (i32) f64)   ; f64.uconvert_i32
    (f64.convert_s/i64 #xb0 (i64) f64)   ; f64.sconvert_i64
    (f64.convert_u/i64 #xb1 (i64) f64)   ; f64.uconvert_i64
    (f64.promote/f32 #xb2 (f32) f64)     ; f64.convert_f32
    (f64.reinterpret/i64 #xb3 (i64) f64) ; f64.reinterpret_i64
    (i64.reinterpret/f64 #xb5 (f64) i64) ; i64.reinterpret_f64
```

**Comment by kripken** [comment]
This might be fixed by e81ed44795bcb3e3c181177009cb6390d1b0a555 on the `binary-spec-tests` branch.

**Comment by JSStats** [comment]
Sorry, this does not appear to fix the issue. See failures for all these opcodes now: 9d 9e 9f a0 a2 a3 a4 a5 a8 a9 aa ab ad ae af b0 b1 b3 b4 b5
